Ecommerce/index.html

```diff
@@ -1,13 +1,83 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>E-commerce</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <!-- Start coding from the lines below for ECOMMERCE TASK -->
-    <!-- to import image from public folder, do it like this = <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
-</html>
+console.log('Hello World');<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to My Shop</title>
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            margin: 0;
+            padding: 0;

```

Comment:
Consider adding more spacing between the 'padding' and 'margin' properties for better readability.